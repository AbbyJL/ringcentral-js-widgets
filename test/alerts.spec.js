import toJson from 'enzyme-to-json';

import wrapper from './shared';

describe('test alerts', () => {
  test('initial state', () => {
    expect(wrapper).toBeDefined();
    expect(toJson(wrapper)).toBeDefined();
  });
});
