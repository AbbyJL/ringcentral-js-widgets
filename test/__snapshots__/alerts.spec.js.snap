// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test alerts initial state 1`] = `undefined`;
